getting-started/templates/systemlink-values.yaml

```diff
@@ -897,3 +897,51 @@ comments:
   ## Maximum number of comments supported per delete comments request.
   ##
   maximumCommentsPerDeleteRequest: 1000
+
+## Configuration for work order service.
+##
+workorder:
+  ## Set to true to enable the deployment of the work order and lab management ui service. By default, 
+  ## this is disabled and the work order and lab management ui service are not deployed.
+  ##
+  enabled: false
+
+  ## Maximum number of work orders per create work orders request.
+  ##
+  maximumWorkOrdersPerCreateRequest: 1000
```

Comment:
Do we expect a typical customer to need to set any of these limits to something other than the default? If not, please do not include them in this file. We should avoid adding unnecessary complexity to this file.

Reply:
The customer would not need to update these limits other than the default. Removed it.